source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -204,41 +232,54 @@ There isn't currently a QIIME 2 function to trim reads to the same length withou
 Quality filtering
 '''''''''''''''''
 
+**Relevant plugin**: :doc:`quality-filter <../plugins/available/quality-filter/index>`
+
 You can perform different types of quality filtering with the :doc:`quality filter <../plugins/available/quality-filter/index>` plugin.
-**TODO: what's the difference between the q-score and q-score-joined? Why are there two separate functions?**
+The `q-score` method is for single- or paired-end sequences (i.e. `SampleData[PairedEndSequencesWithQuality | SequencesWithQuality]`) while `q-score-joined` is for merged reads (i.e. `SampleData[JoinedSequencesWithQuality]`).
```

Comment:
is for merged reads --> is for joined reads

sorry to get pedantic... both terms are used, but we prefer using "joined" paired-end reads to avoid confusion with "merging" e.g., feature tables or sequence files.